src/index.css

```diff
@@ -50,12 +50,17 @@ a {
 .solr-search-results ul label {
   display: inline-block;
   margin: 0 20px 0 0;
-  width: 150px;
+  width: 180px;
   color: #666;
-  overflow: hidden;
+  overflow-y: scroll;
   white-space: nowrap;
   vertical-align: bottom;
-  text-overflow: ellipsis
+}
+
+.solr-search-results ul label::-webkit-scrollbar {
+  width: 0;  /* Remove scrollbar space */
+  height: 0;
+  background: transparent;  /* Optional: just make scrollbar invisible */
```

Comment:
Some fields' labels are too long to be rendered. Add `overflow: scroll` function to view the full labels and hide the scrollbar.